src/main/java/nl/novi/HwLes10ANWMTechItEasy/Models/Television.java

```diff
@@ -0,0 +1,22 @@
+package nl.novi.HwLes10ANWMTechItEasy.Models;
+
+public class Television {
+    private String brand;
+    private String type;
+
```

Comment:
ALs je een model maakt leer je zelf aan om altijd een id toe tevoegen. Dan kun je in je controller ook het id van het object targeten inplaats van de index in de array zoals nu gebeurd.